src/recommendations/implementation.md

```diff
@@ -0,0 +1,27 @@
+---
+group: product-recommendations
+title: Implementation Steps
+ee_only: True
+---
+
+To help you implement product recommendations on your storefront, use the following workflow:
+
+1. **Deploy data collection to production**
```

Comment:
```suggestion
1. **Deploy [data collection]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data) to the production storefront**
```

Reply:
I'd rather not make the heading a link. I've incorporated your change into the Why section.